addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanner.java

```diff
@@ -327,6 +354,9 @@ private String getCSPNoticesString(ArrayList<Notice> notices) {
         if (pol.allowsPrefetchFromSource(PARSED_WILDCARD_URI)) {
             allowedSources.add("prefetch-src");
         }
+        if (pol.allowsFormAction(PARSED_WILDCARD_URI)) {
+            allowedSources.add("form-action");
```

Comment:
Rational? (Why not `base-uri` too?)

Reply:
This was added because I missed it before. base-uri wasn't because salvation didn't have a call for it 😞